app/presenters/publishing_api_presenters.rb

```diff
@@ -27,6 +18,12 @@ def self.presenter_class_for(model)
       PublishingApiPresenters::TakePart
     when Topic
       PublishingApiPresenters::PolicyAreaPlaceholder
+    when ::StatisticsAnnouncement
+      if model.unpublished? || (model.publication && model.publication.published?)
```

Comment:
This seems like more of a presenter concern to me which should be tidied up in a future refactor as per [the commit description](https://github.com/alphagov/whitehall/commit/19bad1c0edb172844bd2a7879ae3dbdb6fa50034) I agree it is ugly at the moment though and am happy to move it. Maybe a `requires_redirect?` method on the object?